Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 ruby '2.3.1'
 
+gem 'cloudinary'
```

Comment:
Better to group gems in the Gemfile together with other gems that are related to the same features/functionality. E.g. carrierwave first, then cloudinary.